4_demo/cloudbuild.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+timeout: 3600s # 1hr
+tags:
+  - modern-cicd-anthos
+  - modern-cicd-anthos-demo-apps
+substitutions:
+  _DOMAIN: ""
+  _BUILD_IMAGES: "true"
+  _APPS: "hipster-loadgenerator hipster-shop hipster-frontend petabank"
+options:
+  substitution_option: 'ALLOW_LOOSE'
+steps:
+- name: 'gcr.io/${PROJECT_ID}/anthos-platform-installer'
+  id: "install-and-configure-demo-apps"
+  entrypoint: bash
+  args:
+    - -c
+    - |
+      # TODO: Convert to gzip
+      # Pull in starter-repos
+      gsutil cp -r gs://${PROJECT_ID}-starter-repos/starter-repos .
+
+      export GITLAB_PASSWORD=$(gcloud secrets versions access latest --secret="gitlab-password")
+      export GITLAB_HOSTNAME="gitlab.${_DOMAIN}"
+      export GITLAB_TOKEN="$${GITLAB_PASSWORD}"
+      export SERVICES="${_APPS}"
```

Comment:
Lets rename this to APPS in the script too so we dont have to do this translation.

Reply:
Will do